src/services/TaskRepository.ts

```diff
@@ -0,0 +1,114 @@
+//import { Tasks } from "../models/ArrayModel";
+export class TaskRepo {
+    getTasks(completion:boolean, sortBy:string, sortOrder:string){
+        var queriedTasks:Task[]
+
+        switch (completion)
+        {
+            case true:
+                queriedTasks = Tasks.filter(Task => Task.completed == true);
+                break;
+            case false:
+                queriedTasks = Tasks.filter(Task => Task.completed == false);
+                break;
+            default:
+                queriedTasks = Tasks;
+        }
```

Comment:
this could be written entirely as

``` 
Tasks.filter(Task => Task.completed == completion)
```

no switch statement required

Reply:
Paired it down, but I do still need to check for existence of a completion parameter. The above returns an empty array if there's no parameter (ie: we want to see **all** tasks).